official-templates/stable-diffusion-webui/Dockerfile

```diff
@@ -48,17 +43,23 @@ RUN mkdir /sd-models && mkdir /cn-models && \
 RUN python -m venv /workspace/venv && \
     pip install -U --no-cache-dir jupyterlab jupyterlab_widgets ipykernel ipywidgets
 
+# Install torch and xformers
+RUN cd /workspace/stable-diffusion-webui && \
+    pip install torch===${TORCH_VERSION} torchvision torchaudio --force-reinstall --index-url https://download.pytorch.org/whl/cu118 && \
+    pip install xformers==${XFORMERS_VERSION}
+
 # Install Automatic1111's WebUI
 RUN git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui.git && \
     cd stable-diffusion-webui && \
     git checkout tags/${WEBUI_VERSION} && \
-    mv /workspace/requirements.txt ./requirements.txt && \
+    pip install -r requirements_versions.txt && \
     python -c "from launch import prepare_environment; prepare_environment()" --skip-torch-cuda-test
 
 # Install DeForum
 RUN cd /workspace/stable-diffusion-webui && \
     git clone https://github.com/deforum-art/sd-webui-deforum extensions/deforum && \
     cd extensions/deforum && \
+    pip install basicsr && \
```

Comment:
basicsr was previously installed as part of A1111 requirements, but has been removed from A1111 requirements in A1111 v1.8.0, and is still a dependency of Deforum